.github/workflows/docker-ingestion-base.yml

```diff
@@ -49,14 +49,14 @@ jobs:
       security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
       actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
     name: "[Monitoring] Scan ingestion-base images for vulnerabilities"
+    if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
Please see https://github.com/datahub-project/datahub/actions/runs/4237278934/jobs/7364447126 I don't think this will solve the problem either.